assets/__tests__/helpers.js

```diff
@@ -0,0 +1,26 @@
+import { JSDOM } from 'jsdom'
+
```

Comment:
Let’s leave these duplicated for now.  Still figuring out the right tests with this.  Some abstractions may become obvious later